.github/workflows/deployment-release.yaml

```diff
@@ -91,6 +91,7 @@ jobs:
           FAIL_ON_REDIS_UNAVAILABLE: true
           AUTH_COOKIE_ENABLED: true
           IDENTITY_TOKEN: ${{ secrets.IDENTITY_TOKEN }}
+          BLOCKNUM_AUTH_ENABLED: true
```

Comment:
Aren't we disabling block number authentication on prod?

Reply:
in this step of the workflow tests are executed and this settings is required only for tests

Reply:
Its will not hurt of course, but tests are ran in ci.yaml already